src/components/WorkFacets/WorkFacets.tsx

```diff
@@ -67,7 +67,7 @@ const WorkFacets: React.FunctionComponent<Props> = ({
 
     return (
       <Link href={url + params.toString()}>
-        <a>
+        <a id={"facet-"+param}>
```

Comment:
I think we need to make these more unique (or use classes).  Since this is called in a loop for each value in the facet, the `a` tags cannot all have the same ID.